README.md

```diff
@@ -96,24 +99,26 @@ update_all(CONFIG_FILE_PATH)
 
 ### CLI
 ⚠️ You can use `dmeta` or `python -m dmeta` to run this program
+⚠️ You can use the `inplace` flag in CLI to indicate if the action should affect the original file(s).
+
 #### Version
 ```console
 dmeta -v
 dmeta --version
 ```
-### Clear metadata for a .docx file
+#### Clear metadata for a .docx file in place
```

Comment:
I'm sorry, I'm not sure if I understood your point, could you please re-explain it?

Reply:
my bad.
I meant, right now all the examples in README (all the above Python scripts) are using `in_place=True`. It's good to have a Python example for `in_place=False` (default mode) too.

Reply:
They are not I guess, in each function, if you do not specify `in_place` it will fallbacks to `False` as the default option.
so the `clear_all` example and the `update_all` example work in `in_place=False` mode.
Good?
![image](https://github.com/user-attachments/assets/2b7d09ed-41ea-44d2-a06f-fa3001ab5fa1)